src/main/java/org/spongepowered/api/text/channel/MessageChannel.java

```diff
@@ -106,6 +107,30 @@ static MessageChannel combined(Collection<MessageChannel> channels) {
     }
 
     /**
+     * Creates a message channel that targets subjects contained within every given
+     * channels and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see IntersectedMessageChannel
+     */
+    static MessageChannel intersected(MessageChannel... channels) {
+        return new IntersectedMessageChannel(channels);
```

Comment:
Maybe this should check if there is only one element in the channel list and return that in this case.